setup.py

```diff
@@ -244,6 +244,7 @@ def extract_from_precompiled(precompiled_location: str, package_data: List[str],
     package_data={
         'tensorrt_llm': package_data,
     },
+    license_files=["LICENSE", "ATTRIBUTIONS-CPP.md", "ATTRIBUTIONS-Python.md"],
```

Comment:
I think ATTRIBUTIONS-Python.md is primarily for the container. Adding it to the wheel would probably be confusing, since we don't include the packages mentioned by ATTRIBUTIONS-Python.md inside the wheel.

```suggestion
    license_files=["LICENSE", "ATTRIBUTIONS-CPP.md"],
```

Reply:
thanks for the catch @tburt-nv , while we have asked for guidance, my lookups also indicate that we need not include it, so have removed from packaging.